### PR TITLE
Decide on channel names, and message fields

### PR DESCRIPTION
**Pull request by tmplt**
Discuss in this PR what channel names we should use and what fields messages sent should include.

### CODE REVIEW

File: src/messages.lcm [hunk 1]
```diff
@@ -1,25 +1,99 @@
...
+                PORT_C = 7,
+                PORT_D = 8;
+}
+// Found on channel IO_ENCODER
+struct io_encoder_t
+{
+        brickpi_t brickpi;
+        port_t port;
+        int64_t dt;
+}
```
Comment (tmplt) [inline src/messages.lcm:53]: Here we also need a counter that holds how many ticks were measured during `dt`.

**Comment by tmplt** [comment]
Requesting review, @WastedWaffle, @Soapmaster.

**Review by jlk122 — APPROVED** [review]
Looks good to me

**Comment by tmplt** [comment]
Before this is merged, a `lcm-debug-spoof-message.py` will be written for debugging purposes. `lcm-sink-print-all.py -> lcm-debug-print-all.py` will have to be updated for all new messages, also.

**Pull request merged**